### ISSUE TITLE
UI improvements

### ISSUE THREAD

**Issue by Bioruebe** [Bioruebe/UniExtract2#126]
- [x] Display changelog in update found dialog
- [x] Progress bar for update search
- [x] Redesign preferences GUI to fit more options
- [x] New file scan result message box: group by detector, make text copyable
- [ ] FFMPEG related GUIs
  - [ ] FFMPEG missing message box: download button, already installed, license agreement link, better texts
  - [ ] Remove from first start assistant, can be downloaded as needed
- [ ] Warn about very small executables, which are unlikely to be extractable at all
- [ ] Warn about web installers
- [ ] Notify if file is in use and cannot be extracted, retry button

**Comment by vatterspun** [comment]
Great stuff.  The original Universal Extractor was great because it was very accessible.  Thanks for spending time on the front and back ends of the program.

**Comment by mirh** [comment]
Since the new UI rolled out (I think.. it's two or three months maybe?) I can no longer drag a file into the "extract" path textbox.

**Comment by vatterspun** [comment]
>Since the new UI rolled out 

Can you detail your setup and what version you're using?

**Comment by mirh** [comment]
~~The info window says I'm using RC2. And I'm on windows 7. 
It shouldn't really be difficult to notice~~

Somehow it works now :/

**Comment by vatterspun** [comment]
> Somehow it works now :/

Yeah, sometimes bugs like to hide.  Maybe it's the dreaded "intermittent" issue and someone will reference your post as evidence.  Anyway, thanks or re-testing.

**Comment by mirh** [comment]
> Somehow it works now :/

Could it be that when you let the program update itself, if you have it launch again automatically, that keeps administrator privilege?
